applications/FluidDynamicsApplication/CMakeLists.txt

```diff
@@ -154,6 +154,9 @@ if(${INSTALL_TESTING_FILES} MATCHES ON )
         PATTERN "*.cpp" EXCLUDE
         PATTERN "*.hpp" EXCLUDE
       )
+
+    install(FILES ${CMAKE_CURRENT_SOURCE_DIR}/tests/CompressibleSymbolicGeneration/compilable_symbolic_test.h 
```

Comment:
By doing this we'd need to manually add any future test. Couldn't we make something similar to what we do above but for a `${CMAKE_CURRENT_SOURCE_DIR}/tests/symbolic_generation` generic folder?

PD: Note that our convention in the repo is to write subfolders in snake case (admittedly not always fulfilled...)

Reply:
Well this was done kind of on purpose. When speaking with @roigcarlo he explained to me that `.h`, `.cpp`, etc. are blacklisted to avoid accidentally poluting the python package. I decided to keep it very restrictive in order to ensure this goal is still fulfilled.

I have no strong feelings one way or the other, just explaining my motivation.

Re: snake case. Not an issue, will change it.